docs/products/kafka/concepts/horizontal-vertical-scaling.rst

```diff
@@ -0,0 +1,50 @@
+Scaling options in Apache Kafka
+===============================
+
+Aiven for Apache Kafka has a number of predefined plans that specify the
+number of brokers and the capacity of individual brokers. The predefined plans consist of 3, 6, 9, 15, or 30 brokers, but we can also create
+larger custom plans based on customer requirements.
+
+To increase the capacity of an existing Kafka cluster, two options are
+available:
+
+* **Vertically** scaling the performance of individual brokers
+
+* **Horizontally** scaling the cluster by adding more brokers
+
+Both scaling options are available for all Aiven for Apache Kafka customers and can be performed as an **online** operation, keeping your cluster up and running during the upgrade. 
+
+.. Note::
+
+    When you change the service plan, Aiven automatically starts adding new brokers with the new specifications to your existing cluster. Once the new brokers are online and the data is replicated to them from the older nodes, the old brokers are retired one by one.
+
+Aiven suggest to utilize both the vertical and horizontal scaling capabilities of Aiven for Apache Kafka to achieve the best possible performance and fault tolerance. 
```

Comment:
I wonder if it is "recommend" and not just "suggest", and I would use "use" instead of "utilise" , but that is my preference.

Reply:
and it is a third person, so "suggests" or "recommends". I also noticed that you use "recommend" a bit later too, so we can use here "advises" instead of "suggests" to give a stronger vibe.

Reply:
fixed, thanks!